docs/source/workflows/predictors.rst

```diff
@@ -462,7 +463,7 @@ This predictor will compute a real descriptor with a key ``mean of property dens
         inputs=[formulation_descriptor]
     )
 
-If ``p`` is given a value other than ``1.0``, that value will be included in the key for the feature, e.g. ``2.0-mean of property viscosity``.
+If ``p`` is given a value other than ``1``, that value will be included in the key for the feature, e.g. ``2.0-mean of property viscosity``.
```

Comment:
If we only support integers we may want to change the string representation so decimals aren't included, e.g. `2-mean of property viscosity` instead of `2.0-mean`.

Reply:
I think it is "2-mean", actually: https://github.com/CitrineInformatics/mithril/blob/master/src/main/scala/io/citrine/nucleus/ml/relations/library/FormulaExpander.scala#L204